syntax = "proto3";

//Req 发
//Res 收

message MsgHeader{
    int32 server_id = 1;
    int32 server_type = 2;
    int64 time_stamp = 3;
}

message Message {
    MsgHeader header = 1;
    bytes content = 2;
    int32 cmd = 3;
}
